arangod/StorageEngine/TransactionState.cpp

```diff
@@ -119,6 +119,7 @@ TransactionState::Cookie::ptr TransactionState::cookie(
 
 /// @brief add a collection to a transaction
 int TransactionState::addCollection(TRI_voc_cid_t cid,
+                                    std::string const& cname,
```

Comment:
why is a name necessary if a cid is already provided?

Reply:
to get rid of the name resolution in TransactionState